dashboard/.example.env

```diff
@@ -1,2 +1,2 @@
-REACT_APP_ENABLE_SOCIAL_AUTH=false
+REACT_APP_ENABLE_SOCIAL_AUTH=true
```

Comment:
@FilipNikolovski should we have this value enabled or should we only manage it locally?